cpp/ql/lib/ext/std.vector.model.yml

```diff
@@ -0,0 +1,59 @@
+extensions:
+  - addsTo:
+      pack: codeql/cpp-all
+      extensible: summaryModel
+    data: # namespace, type, subtypes, name, signature, ext, input, output, kind, provenance
+      - ["std", "vector", True, "assign", "", "", "Argument[*@1]", "Argument[-1].Element[@]", "value", "manual"]
+      - ["std", "vector", True, "assign<InputIt>", "(InputIt,InputIt)", "", "Argument[0]", "Argument[-1].Element[@]", "value", "manual"]
```

Comment:
Should this be from `Argument[0].Element[@]`, since in this case the argument is an iterator?  (same applies for `list`, `forward_list` and `deque`).

Reply:
Indeed, they should! Fixed in 5351c2734f551870fbbe90222af06ea2e71f3787